### DISCUSSION TITLE
Anyone seen this menu scroll issue before?

### DISCUSSION THREAD

**Discussion by SteveRosam** [squidfunk/mkdocs-material#5701] [General]
The nav sometimes appears over the footer and does not scroll.

This is happening intermittently.

![image](https://github.com/squidfunk/mkdocs-material/assets/100689438/d0e53fea-427b-44d8-8d7c-25fd3e9581ff)

**Comment by squidfunk** [comment]
You probably have JavaScript [errors in your console](https://learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/console/console-debug-javascript) that might come from your customizations. Try commenting out `theme.custom_dir` in `mkdocs.yml`. If that fixes the issue, the error lies within your customizations.

If not, we need a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/) to look into it 😊
